### PR TITLE
Add cmake build instructions for Android

### PR DESCRIPTION
**Pull request by OneBadNinja**
Working sndfile integration on Android, used for a simple music player

### CODE REVIEW

File: Building-for-Android.md [hunk 1]
```diff
@@ -15,3 +15,32 @@ by setting them before the script is run.
...
+```
+
+cd into `build` and run make
+
+```
+cd build
+make [-j <number of parallel jobs>]
+```
+
+This will build libsndfile for android
```
Comment (evpobr) [inline Building-for-Android.md:46]: Add dot please.

**Review by SoapGentoo — APPROVED** [review]

File: Building-for-Android.md (hunk 1 above)
Comment (evpobr) [reply Building-for-Android.md:46]: @OneBadNinja ?

**Comment by evpobr** [comment]
Actually i can fix it.

File: Building-for-Android.md (hunk 1 above)
Comment (evpobr) [inline Building-for-Android.md:46]:
```suggestion
This will build libsndfile for android.
```

**Pull request merged**